### ISSUE TITLE
Cannot read properties of undefined (reading `inputPath`) (via TypeError)

### ISSUE THREAD

**Issue by xplosionmind** [photogabble/eleventy-plugin-interlinker#1]
[My website](https://github.com/xplosionmind/tommi.space)’s build fails when adding this plugin:

```
> tommi.space@1.0.0 start
> ELEVENTY_ENV=development eleventy --serve --watch --incremental

[11ty] Problem writing Eleventy templates: (more in DEBUG output)
[11ty] Cannot read properties of undefined (reading 'inputPath') (via TypeError)
[11ty] 
[11ty] Original error stack trace: TypeError: Cannot read properties of undefined (reading 'inputPath')
[11ty]     at compileTemplate (/Users/tommi/tommi.space/node_modules/@photogabble/eleventy-plugin-interlinker/index.js:31:33)
[11ty]     at /Users/tommi/tommi.space/node_modules/@photogabble/eleventy-plugin-interlinker/index.js:147:51
[11ty]     at Array.forEach (<anonymous>)
[11ty]     at /Users/tommi/tommi.space/node_modules/@photogabble/eleventy-plugin-interlinker/index.js:147:14
[11ty]     at Array.forEach (<anonymous>)
[11ty]     at Object.backlinks (/Users/tommi/tommi.space/node_modules/@photogabble/eleventy-plugin-interlinker/index.js:127:16)
[11ty]     at ComputedData._setupDataEntry (/Users/tommi/tommi.space/node_modules/@11ty/eleventy/src/ComputedData.js:98:25)
[11ty]     at ComputedData.processRemainingData (/Users/tommi/tommi.space/node_modules/@11ty/eleventy/src/ComputedData.js:119:16)
[11ty]     at Template.resolveRemainingComputedData (/Users/tommi/tommi.space/node_modules/@11ty/eleventy/src/Template.js:625:31)
[11ty]     at TemplateMap.resolveRemainingComputedData (/Users/tommi/tommi.space/node_modules/@11ty/eleventy/src/TemplateMap.js:731:39)
[11ty] Benchmark   4497ms  50% 14436× (Configuration) "slugify" JavaScript Function
[11ty] Copied 30 files / Wrote 0 files in 6.41 seconds (v2.0.0)
```

[**Full DEBUG log**](https://paste.yunohost.org/raw/yukipeduju.bash)

**Comment by xplosionmind** [comment]
P.S.: I would like to thank you from the bottom of my heart for conceiving this plugin. I love it.

**Comment by carbontwelve** [comment]
Ah, I have just come across this myself on my website:
```
[11ty] Cannot read properties of undefined (reading 'inputPath') (via TypeError)
[11ty] 
[11ty] Original error stack trace: TypeError: Cannot read properties of undefined (reading 'inputPath')
[11ty]     at compileTemplate (/Users/simon/Code/photogabble/website/node_modules/@photogabble/eleventy-plugin-interlinker/index.js:31:33)
[11ty]     at /Users/simon/Code/photogabble/website/node_modules/@photogabble/eleventy-plugin-interlinker/index.js:147:51
[11ty]     at Array.forEach (<anonymous>)
[11ty]     at /Users/simon/Code/photogabble/website/node_modules/@photogabble/eleventy-plugin-interlinker/index.js:147:14
[11ty]     at Array.forEach (<anonymous>)
[11ty]     at Object.backlinks (/Users/simon/Code/photogabble/website/node_modules/@photogabble/eleventy-plugin-interlinker/index.js:127:16)
[11ty]     at ComputedData._setupDataEntry (/Users/simon/Code/photogabble/website/node_modules/@11ty/eleventy/src/ComputedData.js:90:25)
[11ty]     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[11ty]     at async ComputedData.processRemainingData (/Users/simon/Code/photogabble/website/node_modules/@11ty/eleventy/src/ComputedData.js:111:5)
[11ty]     at async Template.resolveRemainingComputedData (/Users/simon/Code/photogabble/website/node_modules/@11ty/eleventy/src/Template.js:625:7)
```

Will be working on a fix today. Appologies for the late reply, it seems Github notifications aren't getting through to my email.

**Comment by xplosionmind** [comment]
No worries @carbontwelve, thanks so much! Will check soon.

**Comment by carbontwelve** [comment]
@xplosionmind this should now be fixed in v1.0.1 by #3. Please reopen this issue if that doesn't fix the problem for you.
